src/index/updater.rs

```diff
@@ -291,7 +291,12 @@ impl Updater {
     outputs_traversed: &mut u64,
   ) -> Result {
     if let Some((ordinal, inscription)) = Inscription::from_transaction(tx, input_ordinal_ranges) {
-      ordinal_to_inscription.insert(&ordinal.n(), &inscription.0)?;
+      // let json = match inscription {
+      // Inscription::Text(content) => serde_json::json!({ "media_type": "text/plain;charset=utf-8", "content": content,}),
+      // Inscription::Png(content) => serde_json::json!({ "media_type": "image/png", "content": content,}),
+      // };
+      let json = serde_json::to_string(&inscription).unwrap();
```

Comment:
I think this unwrap is fine. We should use an `expect` though. "Inscription serialization should always succeed"